_gsdocs-proposals/2020/proposal_CVMFSPodmanIntegration.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Podman CVMFS integration
+layout: gsdocs_proposal
+project: CVMFS
+year: 2019
+organization:
+ - CERN
+---
+
+## Description
+
+[CernVM-FS (CVMFS)][cvmfs] is a globally-distributed filesystem used to lazily distribute software in different server farm.
+
+[Podman][podman] is an utility to run and use containers.
+It provides the same command line interface than Docker and it run without the need of a privileged daemon.
+These two characteristics make it extremely interesting for workload in scientific data-centers.
+
+It has been shown that only a small portion of all the files in a container images is necessary to run the image itself, this is even more accentuated in scientific images since they usually include big binaries, not always necessary.
+Our goal is to merge the lazy load capabilities of CVMFS with the container workflow allowed by podman, to quickly load big scientific images while maintaining the isolation and convenience of containers.
+
+There is already an integration for Docker, and another for containerd (kubernetes) is about ready.
+
+The project will be mentored from both CERN and RedHat.
```

Comment:
You need to add and org file _gsocorgs/2020/redhat.md following the examples in that folder, including a logo REDHAT-logo.png in "images" folder